docs/Supported-OS-Types.md

```diff
@@ -76,13 +76,17 @@
    * [XOS](/lib/oxidized/model/xos.rb)
  * F5
    * [TMOS](/lib/oxidized/model/tmos.rb)
+ * Fiberstore
```

Comment:
How come this is in here?

Reply:
GCOM also produces OEM hardware for other companies
The Fiberstore S3800 is one switch I'm aware of.
I added it so other users can easily find the correct module.

Reply:
Ok, makes sense.